Source/JavaScriptCore/runtime/TemporalPlainTime.h

```diff
@@ -86,6 +86,9 @@ class TemporalPlainTime final : public JSNonFinalObject {
     static std::optional<ISO8601::PlainTime> parse(StringParsingBuffer<CharacterType>&);
     static ISO8601::PlainTime fromObject(JSGlobalObject*, JSObject*);
 
+    ISO8601::Duration differenceTemporalPlainTime(DifferenceOperation,
+        JSGlobalObject*, TemporalPlainTime*, JSValue) const;
```

Comment:
Can you avoid splitting this into two lines?

Reply:
Fixed in [66fe9a7](https://github.com/WebKit/WebKit/pull/50388/commits/66fe9a740f5b5c2e6e103fb24c5924d3fa2b8735)